src/helpers/unmountApplications.ts

```diff
@@ -0,0 +1,10 @@
+import { roots } from '../core/roots';
+import { unmountApplication } from './unmountApplication';
+
+export function unmountApplications()
```

Comment:
Could we rename this to something more explicit? I was originally confused as to why we were unmounting all applications on init of a single application and had to dig into the implementation to discover that it's actually only unmounting applications flagged for unmount.